internal/trace/mock_span.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+
+	"go.opentelemetry.io/api/core"
+	"go.opentelemetry.io/api/tag"
+	apitrace "go.opentelemetry.io/api/trace"
+)
+
+type MockSpan struct {
+	sc     core.SpanContext
+	tracer apitrace.Tracer
+}
+
+var _ apitrace.Span = (*MockSpan)(nil)
+
+// SpancContext returns an invalid span context.
```

Comment:
The comment is false. Also `s/SpancContext/SpanContext/`.

Reply:
fixed.